src/test/java/dog/pawbook/logic/commands/EditOwnerCommandTest.java

```diff
@@ -36,74 +22,6 @@
-        expectedModel.updateFilteredEntityList(new IdMatchPredicate(ID_FIRST_OWNER));
-
-        assertCommandSuccess(editOwnerCommand, model, expectedMessage, expectedModel);
-    }
-
-    @Test
-    public void execute_someFieldsSpecifiedUnfilteredList_success() {
-        Owner toEditOwner = (Owner) model.getEntity(ID_THIRD_OWNER);
-
-        OwnerBuilder ownerInList = new OwnerBuilder(toEditOwner);
-        Owner editedOwner = ownerInList.withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
-                .withTags(VALID_TAG_HUSBAND).build();
-
-        EditOwnerDescriptor descriptor = new EditOwnerDescriptorBuilder().withName(VALID_NAME_BOB)
-                .withPhone(VALID_PHONE_BOB).withTags(VALID_TAG_HUSBAND).build();
-        EditOwnerCommand editEntityCommand = new EditOwnerCommand(ID_THIRD_OWNER, descriptor);
-
-        String expectedMessage = String.format(EditOwnerCommand.MESSAGE_EDIT_OWNER_SUCCESS, editedOwner);
-
-        Model expectedModel = new ModelManager(new Database(model.getDatabase()), new UserPrefs());
-        expectedModel.setEntity(ID_THIRD_OWNER, editedOwner);
-        expectedModel.updateFilteredEntityList(new IdMatchPredicate(ID_THIRD_OWNER));
-
-        assertCommandSuccess(editEntityCommand, model, expectedMessage, expectedModel);
-    }
-
-    @Test
-    public void execute_noFieldSpecifiedUnfilteredList_success() {
-        EditOwnerCommand editEntityCommand = new EditOwnerCommand(ID_FIRST_OWNER, new EditOwnerDescriptor());
-        Owner editedOwner = (Owner) model.getEntity(ID_FIRST_OWNER);
-
-        String expectedMessage = String.format(EditOwnerCommand.MESSAGE_EDIT_OWNER_SUCCESS, editedOwner);
-
-        Model expectedModel = new ModelManager(new Database(model.getDatabase()), new UserPrefs());
-        expectedModel.updateFilteredEntityList(new IdMatchPredicate(ID_FIRST_OWNER));
-
-        assertCommandSuccess(editEntityCommand, model, expectedMessage, expectedModel);
-    }
-
-    @Test
-    public void execute_duplicateOwnerUnfilteredList_failure() {
-        Owner firstOwner = (Owner) model.getEntity(ID_FIRST_OWNER);
-        EditOwnerDescriptor descriptor = new EditOwnerDescriptorBuilder(firstOwner).build();
-        EditOwnerCommand editEntityCommand = new EditOwnerCommand(ID_SECOND_OWNER, descriptor);
-
-        assertCommandFailure(editEntityCommand, model, Messages.MESSAGE_DUPLICATE_OWNER);
-    }
-
-    @Test
-    public void execute_invalidOwnerIdUnfilteredList_failure() {
-        Integer outOfBoundId = model.getUnfilteredEntityList().stream()
-                .map(Pair::getKey).sorted().collect(toList()).get(model.getUnfilteredEntityList().size() - 1) + 1;
-        EditOwnerDescriptor descriptor = new EditOwnerDescriptorBuilder().withName(VALID_NAME_BOB).build();
-        EditOwnerCommand editEntityCommand = new EditOwnerCommand(outOfBoundId, descriptor);
-
-        assertCommandFailure(editEntityCommand, model, Messages.MESSAGE_INVALID_OWNER_ID);
-    }
```

Comment:
Um, I thought u told me to remove all unfilteredList related stuff in the comment from my previous PR?

Reply:
If you want, I can put them back

Reply:
Sorry I didn't make myself clear, I meant to drop the UnfilteredList portion in the name, not to drop the integration tests themselves. Previously there was both UnfilteredList and FilteredList versions of these tests. I have removed the FilteredList versions that's why you see the UnfilteredList ones remaining. It will be good to write similar tests for the other 2 classes as well.